cmd/juju/cloud/updatecloud.go

```diff
@@ -150,6 +150,9 @@ func (c *updateCloudCommand) Run(ctxt *cmd.Context) error {
 		}
 		ctxt.Infof(successMsg)
 	}
+	if c.isPublicCloud(c.Cloud) {
+		ctxt.Infof("To update public clouds from a canonical source, use `juju update-public-clouds`.")
```

Comment:
Not sure users will know what "canonical source" means.
Can we reword to say something like
To ensure this client's copy or any controller copies of public cloud information is up to date with the latest region information, use `juju update-public-clouds`.